NEWS.md

```diff
@@ -26,7 +26,7 @@ MXNet Change Log
       * [CUDA Graphs](#cuda-graphs)
       * [CUDA 11 Support](#cuda-11-support)
       * [TensorRT](#tensorrt)
-      * [OneDNN](#onednn)
+      * [oneDNN](#onednn)
```

Comment:
I am not sure if "Change Log" should be changed. I suppose not

Reply:
As I am not changing any information I believe this small change is fine. @szha Should I leave NEWS.md file without any changes?